static/docs/user-guide/install/linux.md

```diff
@@ -0,0 +1,100 @@
+# Installation on Linux
+
+## Install with pip
+
+```dvc
+$ pip install dvc
+```
+
+Depending on the type of the
+[remote storage](/doc/user-guide/external-dependencies) you plan to use, you
+might need to install optional dependencies: `[s3]`, `[ssh]`, `[gs]`, `[azure]`,
+and `[oss]`. Use `[all]` to include them all.
+
+<details>
+
+### Example: How to install DVC with support for Amazon S3 storage
+
+```dvc
+$ pip install 'dvc[s3]'
+```
+
+In this case it installs `boto3` library as well, besides DVC.
+
+</details>
+
+## Install with conda
+
+> Requires [Anaconda Distribution](https://docs.anaconda.com/anaconda/).
```

Comment:
would be great to mention that it requires Miniconda (the package manager) https://docs.conda.io/projects/continuumio-conda/en/latest/user-guide/install/macos.html (find the right link per platform) or Anaconda. Anaconda is scary to be honest.

Reply:
> would be great to mention that it requires Miniconda (the package manager)

Maybe someone else knows better than me about anaconda/minicoda. I have never used them and whatever I recommend I doubt that might be wrong.

Reply:
Sure! I'll take care of this when it's merged.